### ISSUE TITLE
[New Mod Idea] "Item Crafted/Found By ..."

### ISSUE THREAD

**Issue by BirdieMiner** [ricksouth/serilum-mc-mods#2482]
## **Information**
Minecraft version: 1.20.4
Modloader: Forge

## **Feature description**
I've done a bit of research, and it all seems a bit beyond my understanding, but it would be cool if items had in the description the name of the player that crafted/found them. Crafting tables can't deal with NBT tags apparently, but maybe there could be an inventory check for existing tags, and add 'Crafted By:' to any that are missing? 'Found By' might be when picking up items or generating loot from drops or chests? Not sure what events there are, or if NBT can be read/modified during them though? 

Maybe just unstackable items so it doesn't mess up inventories?

Or maybe just an all-encompassing 'From Player X' or something for anything stored a player's inventory? That might be simpler to implement lol. Enchanting or anvils might be possible directly if they can use NBT data? Maybe a 'Created By Player X' tag added then?

Basically, it would be cool to see where items came from, specifically from which player (for a multiplayer setting), but I honestly have no idea if it's possible, I couldn't find any mods that do this yet

**Comment by ricksouth** [comment]
I appreciate the new mod suggestion! I have added your idea to the list. I'm keeping track of all ideas on the Trello page and added the URL of this issue there.

In order to reduce the amount of open issues on this Github repository, I'll be closing this issue.

It definitely does not mean I am not interested and I'll post a comment here if/when the mod is released. This might take some time, as bug reports take priority. But I'll definitely be releasing more mods along the way.

Thank you very much for the suggestion!
